basculehttp/metricListener.go

```diff
@@ -57,90 +51,32 @@ type MetricListenerOptionsIn struct {
 	Options  []Option `group:"bascule_metric_listener_options"`
 }
 
-// LeewayIn is an uber fx wired struct that provides a bascule leeway, which can
-// be parsed into an Option.
-type LeewayIn struct {
-	fx.In
-	L bascule.Leeway `name:"jwt_leeway" optional:"true"`
-}
-
 // OnAuthenticated is called after a request passes through the constructor and
 // enforcer successfully.  It updates various metrics related to the accepted
 // request.
 func (m *MetricListener) OnAuthenticated(auth bascule.Authentication) {
-	now := time.Now()
-
-	if m.measures == nil {
-		return // measure tools are not defined, skip
-	}
-
+	outcome := AcceptedOutcome
+	// this is weird and we should take note of it.
```

Comment:
I agree. We would only run into this if some middleware is misconfigured in code, right?

Reply:
yeah 👍